docs/pages/more/expo-cli.mdx

```diff
@@ -507,6 +507,7 @@ From here, you can choose to generate basic project files like:
 | `EXPO_NO_TELEMETRY_DETACH`          | **boolean** | <div className="flex items-center pb-1.5"><StatusTag status="SDK 51+" /></div>Send telemetry events in the main thread of `@expo/cli`. This causes the CLI to slow down as it waits for all the events to be sent.                                                                                                                                                                                                              |
 | `EXPO_UNSTABLE_ATLAS`               | **boolean** | <div className="flex items-center pb-1.5"><StatusTag status="experimental" /><StatusTag status="SDK 51+" /></div>Gather Metro bundle information during development or export.                                                                                                                                                                                                                                                  |
 | `EXPO_NO_BUNDLE_SPLITTING`          | **boolean** | <div className="flex items-center pb-1.5"><StatusTag status="experimental" /><StatusTag status="SDK 51+" /></div>Disable Metro splitting chunks on async imports in production (web-only).                                                                                                                                                                                                                                      |
+| `EXPO_USE_METRO_REQUIRE`            | **boolean** | <div className="flex items-center pb-1.5"><StatusTag status="SDK 52+" /></div>Enable the use of Expo's custom Metro `require` implementation and `string` based module IDs. This enables better debugging and deterministic IDs for React Server Components. Does not support legacy RAM bundles.                                                                                                                              |
```

Comment:
```suggestion
| `EXPO_USE_METRO_REQUIRE`            | **boolean** | <div className="flex items-center pb-1.5"><StatusTag status="SDK 52+" /></div>Enable the use of Expo's custom Metro `require` implementation and `string` based module IDs. This enables better debugging and deterministic IDs for React Server Components. Does not support legacy RAM bundles.                                                                                                                               |
```

Apologies for breaking the docs CI in my previous suggestion. I ran these changes locally to fix the Prettier formatting issue here, and I think this suggestion should fix it.